pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = "poetry.masonry.api"
 [tool.black]
 line-length = 88
 target-version = [ "py36", "py37", "py38",]
-exclude = "((.eggs | .git | .pytype | .pytest_cache | build | dist))"
+exclude = "((.eggs | .git | .pytest_cache | build | dist))"
```

Comment:
do we also need to exclude the `mypy` cache here?

Reply:
there is no need, since mypy cache contains json files instead python files (which was the case for pytype)